near/src/config.rs

```diff
@@ -417,11 +417,16 @@ pub struct AccountInfo {
     pub amount: Balance,
 }
 
+pub const CONFIG_VERSION: u32 = 1;
+
 /// Runtime configuration, defining genesis block.
 #[derive(Serialize, Deserialize, Clone, Debug)]
 pub struct GenesisConfig {
     /// Protocol version that this genesis works with.
     pub protocol_version: u32,
+    /// This is a version of a genesis config structure.
```

Comment:
Should clarify that this is the version of the config this version of binary works with. If it tries to load a JSON config with a different version it will panic.